resources/public/api/conf/4.0/self_employment_retrieve.yaml

```diff
@@ -1,35 +1,58 @@
+    | DYNAMIC_SELF_EMPLOYMENT_UNADJUSTED        | Simulates success response with unadjusted summary.                         <br/><br/> See below for details about DYNAMIC scenarios. |                
+    | SELF_EMPLOYMENT_STATUS_INVALID            | Simulates success response with summary status invalid.                                                                               |
+    | DYNAMIC_SELF_EMPLOYMENT_STATUS_INVALID    | Simulates success response with summary status invalid.                     <br/><br/> See below for details about DYNAMIC scenarios. |                    
+    | SELF_EMPLOYMENT_STATUS_SUPERSEDED         | Simulates success response with summary status superseded.                                                                            |
+    | DYNAMIC_SELF_EMPLOYMENT_STATUS_SUPERSEDED | Simulates success response with summary status superseded.                  <br/><br/> See below for details about DYNAMIC scenarios. |                       
+    | NOT_SELF_EMPLOYMENT                       | Simulates the scenario where the calculationId is not a self-employment.                                                              |
+    | TAX_YEAR_NOT_SUPPORTED                    | Simulates the scenario where the tax year is not supported.                                                                           |
+
+    In each case where a scenario is prefixed by DYNAMIC_, the following response values will change based on the values submitted in the request:
+    - metadata
+      - calculationId
+      - taxYear
+      - nino
+      - requestedDateTime
+      - adjustedDateTime
+    - inputs
+      - accountingPeriodStartDate
+      - accountingPeriodEndDate
+      - submissionPeriods
+        - startDate
+        - endDate
+    
+    For tax years prior to 2023-24, when the tax year is not supplied, these values will be based on a tax year of 2022-23.
 
   tags:
     - Self-employment business
 
   parameters:
-    - $ref: ./common/pathParameters.yaml#/components/parameters/nino
-    - $ref: ./common/pathParameters.yaml#/components/parameters/calculationId
-    - $ref: ./common/queryParameters.yaml#/components/parameters/taxYearTys
-    - $ref: ./common/headers.yaml#/components/parameters/acceptJson
-    - $ref: ./common/headers.yaml#/components/parameters/authorizationReadScopeHeader
-    - $ref: ./common/headers.yaml#/components/parameters/testScenarioHeader
+    - $ref: "./common/pathParameters.yaml#/components/parameters/nino"
+    - $ref: "./common/pathParameters.yaml#/components/parameters/calculationId"
+    - $ref: "./common/queryParameters.yaml#/components/parameters/taxYearTys"
+    - $ref: "./common/headers.yaml#/components/parameters/acceptJson"
+    - $ref: "./common/headers.yaml#/components/parameters/authorizationReadScopeHeader"
+    - $ref: "./common/headers.yaml#/components/parameters/testScenarioHeader"
```

Comment:
I added the quotes because IntelliJ was having trouble understanding some of these links. Now we should be able to CMD+Click to get to the definitions and check for broken links.

![image](https://github.com/hmrc/self-assessment-bsas-api/assets/141128938/2e71e080-d271-4fa6-bd88-cb048ea3fab2)